packages/kbn-test-eui-helpers/src/rtl_helpers.tsx

```diff
@@ -9,7 +9,7 @@ import moment from 'moment';
 import userEvent from '@testing-library/user-event';
 import { screen, within, fireEvent } from '@testing-library/react';
 
-export const getButtonGroupInputValue = (testId: string) => () => {
```

Comment:
I didn't like this name as the function returns the selected button component and not its value.